### PR TITLE
Adding LVS environment for GF180MCU

### PR DESCRIPTION
**Pull request by FaragElsayed2**
## Adding LVS environment for GF180MCU

### CODE REVIEW

File: klayout/lvs/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# LVS Documentation
+
+Explains how to use the runset.
+
+## Folder Structure
+
+```text
+📦runset
```
Comment (atorkmabrains) [inline klayout/lvs/README.md:8]: @FaragElsayed2 Please change the icon to folder as requested by proppy.

File: klayout/lvs/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# LVS Documentation
```
Comment (atorkmabrains) [inline klayout/lvs/README.md:1]: @FaragElsayed2 Please update the LVS documentation.

**Review by atorkmabrains — CHANGES_REQUESTED** [review]

File: klayout/lvs/testing/Makefile [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+.ONESHELL:
+test-LVS-switch:
+	@cd $(Testing_DIR)
+	@cd ..
+	@echo "========== LVS-Switch testing =========="
+	@python3 run_lvs.py --design=testing/extraction_checking/sample_nfet_03v3.gds --net=sample_nfet_03v3.spice --gf180mcu=A 
+	@python3 run_lvs.py --design=testing/extraction_checking/sample_nfet_03v3.gds --net=sample_nfet_03v3.spice --gf180mcu=B
+	@python3 run_lvs.py --design=testing/extraction_checking/sample_nfet_03v3.gds --net=sample_nfet_03v3.spice --gf180mcu=C
+
+#==========================
```
Comment (FaragElsayed2) [inline klayout/lvs/testing/Makefile:52]: `Help section` To be removed

**Comment by atorkmabrains** [comment]
@FaragElsayed2 Remove fab/man test cases and use a single folder for unit testing and avoid complexity.

**Comment by atorkmabrains** [comment]
@FaragElsayed2 split the lvs file into sections and include each in the main LVS file.

**Comment by atorkmabrains** [comment]
Inside the test cases, we need the following folder structure:
```
--testcases
|---->unit
|       |----> group
|                 |-----> layout
|                 |-----> netlist
|.....
```

File: klayout/lvs/README.md (hunk 1 above)
Comment (FaragElsayed2) [reply klayout/lvs/README.md:8]: Done

File: klayout/lvs/README.md (hunk 2 above)
Comment (FaragElsayed2) [reply klayout/lvs/README.md:1]: Done

File: klayout/lvs/testing/Makefile (hunk 3 above)
Comment (FaragElsayed2) [reply klayout/lvs/testing/Makefile]: Done

**Comment by FaragElsayed2** [comment]
@atorkmabrains 

- LVS runset has been spliited into sections as agreed 
- New folder structure is added and testcases have been fixed
- Regression is changed and could be run multi-threaded now with more detailed results.


Still need to add switches for devices that need it to pass CI

**Pull request closed without merging**